src/main/java/lz/modelTesting/controllers/MockupsController.java

```diff
@@ -76,8 +77,17 @@ private Mockup createMockupFromRequest(HttpServletRequest request) {
         String src = request.getParameter(SOURCE_LINK);
         String projectId = request.getParameter(PROJECT_ID);
         Mockup mockup = new Mockup(name, src, projectId);
-        mockupsRepository.save(mockup);
+
         return mockup;
     }
 
+    @PostMapping(value = "/archive/{id}")
+    public ResponseEntity<String> addMockup(HttpServletRequest request,@PathVariable String id) {
```

Comment:
This function should be named `archiveMockup`

Reply:
Add whitepace between args